packages/ui/src/core/tamagui.ts

```diff
@@ -16,6 +16,8 @@ export {
   TextArea,
   Text,
   Dialog,
+  Popover,
```

Comment:
Can we scratch these for now? Just trying to limit the higher-level tamagui we stuff we add because not all of it is super baked.